### PR TITLE
make all packages serviceable by default

### PR DESCRIPTION
**Pull request by anurse**
fixes https://github.com/dotnet/cli/issues/2569

/cc @pakrym @davidfowl @ericstj

---
This change is [Reviewable](https://reviewable.io/reviews/dotnet/cli/2575)

### CODE REVIEW

**Comment by pakrym** [comment]
:shipit:

File: src/Microsoft.DotNet.ProjectModel/DependencyContextBuilder.cs [hunk 1]
```diff
@@ -115,7 +115,9 @@ private static CompilationOptions GetCompilationOptions(CommonCompilerOptions co
         {
             var type = export.Library.Identity.Type;
 
-            var serviceable = (export.Library as PackageDescription)?.PackageLibrary.IsServiceable ?? false;
+            // TEMPORARY: All packages are serviceable in RC2
+            // See https://github.com/dotnet/cli/issues/2569
+            var serviceable = (export.Library as PackageDescription) != null;
```
Comment (gkhanna79) [inline src/Microsoft.DotNet.ProjectModel/DependencyContextBuilder.cs]: Why didnt we just set it to true?

Comment (anurse) [inline src/Microsoft.DotNet.ProjectModel/DependencyContextBuilder.cs]: Because I don't want to set it on Project dependencies.

**Comment by anurse** [comment]
@dotnet-bot test OSX x64 Release Build please

**Pull request merged**